4-AccessControl/1-app-roles/App/package-lock.json

```diff
@@ -25,11 +25,11 @@
         "supertest": "^6.1.4"
       }
     },
-    "../../../common/msal-node-wrapper": {
```

Comment:
How about you delete the package-lock.json? This way the sample will always use the latest version of MSAL.

Reply:
Oh, shoot! I misread the GitHub Issue connected to this PR. I'll delete them.

Reply:
The build scripts fail when there is no package-lock.json

Reply:
Can you fix them?

Reply:
Per offline discussion, this will be a separate task.